CHANGELOG.md

```diff
@@ -5,7 +5,10 @@ Version [2.2dev] - XXXX-XX-XX
 -------------------------------------------------------------------------------
```

Comment:
As discussed at a previous meeting, this will be version 3.0, so we should address this in the changelog (and put the merge date in it)

Reply:
Resolved in 3fc1032